.github/actions/e2e_recover/action.yml

```diff
@@ -45,7 +47,7 @@ runs:
                 echo "$output"
                 i=$(echo "$output" | grep -o "Pushed recovery key." | wc -l | sed 's/ //g')
                 recovered=$((recovered+i))
-                if [[ $recovered -eq ${{ inputs.controlNodesCount }} ]]; then
+                if [[ $recovered -gt ${{ inputs.controlNodesCount }}/2 ]]; then
                     break
```

Comment:
I also thought and tested that but it doesn't work. I talked to @malt3 about it, and he confirmed that etcd needs a quorum to fully restart. Before it seems like that the kms pod doesn't come up.

Reply:
If the join-service doesn't function as longs as n/2 + 1 nodes are available, then this is fine.
All I went by is that recovering at least one node theoretically brings back Kubernetes.
I guess in the past we also mostly saw failing recover tests with a single node missing, so that issue should be fixed by this